code/modules/unit_tests/_unit_tests.dm

```diff
@@ -27,6 +27,7 @@
 #include "quick_swap_sanity.dm"
 #include "reagent_id_typos.dm"
 #include "reagent_recipe_collisions.dm"
+#include "reagent_mod_procs.dm"
```

Comment:
Alphabetical order--put this before recipe_collisions.

Reply:
yes sir,